manifests/s/Splashtop/SplashtopPersonal/3.5.0.3/Splashtop.SplashtopPersonal.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Splashtop.SplashtopPersonal
+PackageVersion: 3.5.0.3
+PackageLocale: en-US
+Publisher: Splashtop Inc.
+PackageName: Splashtop Personal - Remote Desktop for Windows

```

```suggestion
PackageName: Splashtop Personal
```
Must match control panel

---

changed, thanks